isis/src/control/objs/ControlNet/ControlNet.h

```diff
@@ -221,6 +237,25 @@ namespace Isis {
       friend class ControlPoint;
 
     public:
+      struct Image {
+        QString serial;
+        QList<ControlMeasure* > measures;
+      };
+
+      struct Connection {
+        int strength = 0;
+      };
+
+      // typedefs to help cut down on templated type bloat
+      typedef boost::adjacency_list<boost::setS, boost::listS, boost::undirectedS, Image, Connection> Network;
+      typedef Network::vertex_descriptor ImageVertex;
+      typedef Network::edge_descriptor ImageConnection;
+      typedef std::map<ImageVertex, size_t> VertexIndexMap;
+      typedef boost::associative_property_map<VertexIndexMap> VertexIndexMapAdaptor;
+      typedef Network::out_edge_iterator ConnectionIterator;
```

Comment:
The implementation of the graph is private, so the bundled property structs and the typedefs should all be private

Reply:
:+1: Done.